test/tests/TestRemoveSymlinks.hx

```diff
@@ -0,0 +1,41 @@
+package tests;
+
+import haxe.unit.TestCase;
+
+class TestRemoveSymlinks extends TestCase
+{
+	//----------- properties, fields ------------//
+
+	//--------------- constructor ---------------//
+	public function new()
+	{
+		super();
+	}
+
+
+	//--------------- initialize ----------------//
+
+	override public function setup():Void
+	{
+		Sys.command("neko", ["./bin/haxelib.n", "setup", Sys.getCwd() + "haxelib-repo"]);
+		Sys.command("neko", ["./bin/haxelib.n", "install", "nme-dev", "1.3.2", "--always"]);
```

Comment:
Well, I think using something smaller than nme-dev should not be that much of an issue, e.g. mcli, or whetever else you find under 100KB.